# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../lua-lanes/lua-lanes-2.0.10-destdir-prefix.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- lanes-2.0.10/Makefile.orig	2011-03-11 15:48:00.926696388 +0100
+++ lanes-2.0.10/Makefile	2011-03-11 15:52:34.530696290 +0100
@@ -177,17 +177,18 @@
 #
 # LUA_LIBDIR and LUA_SHAREDIR are used by the .rockspec (don't change the names!)
 #
-DESTDIR=/usr/local
-LUA_LIBDIR=$(DESTDIR)/lib/lua/5.1
-LUA_SHAREDIR=$(DESTDIR)/share/lua/5.1
+DESTDIR=
+PREFIX=/usr/local
+LUA_LIBDIR=$(PREFIX)/lib/lua/5.1
+LUA_SHAREDIR=$(PREFIX)/share/lua/5.1
 
 #
 # AKa 17-Oct: changed to use 'install -m 644' and 'cp -p' 
 #
 install: $(_TARGET_SO) src/lanes.lua
-	mkdir -p $(LUA_LIBDIR) $(LUA_SHAREDIR)
-	install -m 644 $(_TARGET_SO) $(LUA_LIBDIR)
-	cp -p src/lanes.lua $(LUA_SHAREDIR)
+	mkdir -p $(DESTDIR)$(LUA_LIBDIR) $(DESTDIR)$(LUA_SHAREDIR)
+	install -m 644 $(_TARGET_SO) $(DESTDIR)$(LUA_LIBDIR)
+	cp -p src/lanes.lua $(DESTDIR)$(LUA_SHAREDIR)
 
 
 #--- Packaging ---
